modules/mcagar/src/main/ts/ephox/mcagar/api/TinyLoader.ts

```diff
@@ -46,8 +46,11 @@ const setupFromElement = (callback: Loader.RunCallback, settings: Record<string,
   }, settings, Option.some(element));
 };
 
+const removeTinymceElements: () => void = Loader.removeTinymceElements;
```

Comment:
This shouldn't be exported. I've looked where you're using it and you should be using `Editor` from McAgar instead (I'll leave a comment there with more details).

Reply:
Fixed.